pkg/virt-config/config-map.go

```diff
@@ -60,7 +60,7 @@ const (
 	MigrationCompletionTimeoutPerGiB         int64  = 800
 	DefaultMachineType                              = "q35"
 	DefaultCPURequest                               = "100m"
-	DefaultMemoryRequest                            = "8Mi"
+	DefaultMemoryOvercommit                         = 100
```

Comment:
In general I like this setting. Is this something which oVirt has too?

Reply:
Yes, there is such setting of [memory-overcommit at the cluster level](https://github.com/oVirt/ovirt-engine/blob/master/backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/businessentities/Cluster.java#L55) in oVirt that the default minimum allocated memory of the VM (that is used for scheduling) is derived from.